src/main/java/sql/CrudRealization.java

```diff
@@ -0,0 +1,96 @@
+package sql;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class CrudRealization {
```

Comment:
Please use one Crud repository per entity. Your code violates SOLID principles.
https://careers.epam.by/content/dam/epam/by/book_epam_by/JAVA_Methods_Programming_v2.march2015.pdf
page 350. It would help you.